py/visdom/server.py

```diff
@@ -382,7 +386,7 @@ def on_message(self, message):
                 logging.info('closing window {}'.format(msg['data']))
                 p_data = self.state[msg['eid']]['jsons'].pop(msg['data'], None)
                 event = {
-                    'event_type': 'Close',
+                    'event_type': 'close',
```

Comment:
Note - this fixed a bug where close events weren't being picked up on the frontend due to capitalization